snippets/facets.liquid

```diff
@@ -157,14 +169,21 @@
                       <fieldset class="facets-wrap parent-wrap {% if filter_type == 'vertical' %} facets-wrap-vertical{% endif %}">
                         <legend class="visually-hidden">{{ filter.label | escape }}</legend>
                         <ul
-                          class="{% if filter_type != 'vertical' %} facets__list{% endif %} list-unstyled no-js-hidden"
+                          class="{{ visual_layout_class }}{% if filter_type == 'vertical' %} facets__list--vertical{% else %} facets__list{% endif %} list-unstyled no-js-hidden"
                           role="list"
                         >
                           {%- for value in filter.values -%}
+                            {% liquid
+                              assign is_disabled = false
+                              if value.count == 0 and value.active == false
+                                assign is_disabled = true
+                              endif
+                            %}
```

Comment:
I think you can just do:

```liquid
assign is_disabled = value.count == 0 and value.active == false
```

(and if so, same thing down on line ~715)

Reply:
Ahh right, liquid doesn't support assigning via ternaries, so I guess that extends to assigning _any_ expression, including a boolean one.

Sorry, my liquid is pretty rusty 🙈